pyanaconda/ui/tui/spokes/installation_source.py

```diff
@@ -116,22 +116,20 @@ def _on_payload_failed(self):
 
     @property
     def status(self):
-        if self._error:
-            return _("Error setting up software source")
-        elif not self.ready:
-            return _("Processing...")
-        elif not self.payload.is_complete():
-            return _("Nothing selected")
-        else:
-            source_proxy = self.payload.get_source_proxy()
-            return source_proxy.Description
+        """The status of the spoke."""
+        if not self.ready:
+            return _(PAYLOAD_STATUS_SETTING_SOURCE)
+
+        if not self.completed:
+            return _(PAYLOAD_STATUS_INVALID_SOURCE)
+
+        source_proxy = self.payload.get_source_proxy()
+        return source_proxy.Description
 
     @property
     def completed(self):
-        if flags.automatedInstall and self.ready and not self.payload.is_ready():
```

Comment:
Yay, less flags! But how can you just stop using it? It isn't obvious what is different now. Perhaps it would be good to explain that somewhere? Otherwise it looks like just removing things randomly.

Reply:
I couldn't figure out why these checks should be different for automated installations and the commit messages didn't provide any clues. At the end, we should care about only three things:

1. Nothing is happening with the payload at this moment (`self.ready`).
2. The latest payload setup didn't fail with an error (`self._error`).
3. The payload has some usable repositories enabled (`self.payload.is_ready()`).

Could you explain what is missing, please?

Reply:
That sounds great! So maybe the missing bit is where this could be in the commit message?

Reply:
Updated.